src/estimation/include/iDynTree/Estimation/SimpleLeggedOdometry.h

```diff
@@ -249,11 +267,16 @@ class SimpleLeggedOdometry
      * \note this method works only for link, not for arbitrary frames.
      */
     iDynTree::Transform getWorldLinkTransform(const LinkIndex frame_index);
+
+    /**
+     * Get the world_H_link transform for an arbitrary link.
+     *
+     * \note this method works also for arbitrary frames.
+     */
+    iDynTree::Transform getWorldFrameTransform(const LinkIndex frame_index);
```

Comment:
Maybe this method can be turned into a ``const`` method. Again, ``const`` in the parameter is not necessary.